src/aap_eda/settings/default.py

```diff
@@ -478,3 +478,17 @@ def _get_secret_key() -> str:
 ANSIBLE_BASE_JWT_KEY = settings.get(
     "ANSIBLE_BASE_JWT_KEY", "https://localhost"
 )
+
+# TODO: fix to get db address for activation
+ACTIVATION_DB_HOST = "host.containers.internal"
```

Comment:
@Alex-Izquierdo How do we solve this for the installer, operator, Minikube and podman. The current setting works for podman. For minikube or operator it might have to be a different value. We should track this separately.

Reply:
@hsong-rh I think this should be
```
ACTIVATION_DB_HOST = settings.get("ACTIVATION_DB_HOST", "host.containers.internal")
```

Since we default the deployment to podman this would align well and in the file https://github.com/ansible/eda-server/blob/rework-activation/tools/deploy/environment.properties.example we add an entry for K8S to indicate that a value is needed for minikube. Not sure what that default value for ACTIVATION_DB_HOST should be.
@msmagnanijr this might also effect the work you are doing for the operator based deployment

Reply:
This hostname is going to be used for both, event_streams and activations, right? I feel the name might be confusing. 

I see multiple ways to handle it. 

For custom values of PG_NOTIFY_DSN we must state that the hostname must be reachable and resolvable by the containers. 

For our default we could handle it based on the value provided in DATABASES. Probably not even we need a new variable for this. I think the logic can be simple: we only want "host.containers.internal" when the deployment type is podman and the database is localhost. For k8s this value would be never localhost but a service or external.